calico.yaml

```diff
@@ -0,0 +1,371 @@
+package:
+  name: calico
+  version: 3.25.1
+  epoch: 3
+  description: "Cloud native networking and network security"
+  target-architecture:
+    - x86_64
+    - aarch64
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - bash
+      - go
+      # for felix
+      - elfutils-dev
+      - zlib-dev
+      - iproute2
+      - zstd-dev
+      - llvm15
+      - clang-15
+      - libbpf
+      - linux-headers
+      # for BIRD
+      - flex
+      - bison
+      - autoconf
+      - automake
+      # TODO: Can remove this when cni-plugins are built from source
+      - curl
+
+# The calico build process is heavily tailored towards using Makefiles to
+# orchestrate docker commands. This makes for a less than ideal
+# experience replicating it in melange. To try to maintain some level of
+# sanity while packaging and updating calico, the approach here choses to
+# use the upstream Makefile whenever possible, and only when necessary,
+# use the equivalent evaluated `docker run ...` commands within
+# independent pipeline blocks. Throughout this file you'll see the relevant
+# comments denoting the equivalent Makefile targets that each pipeline
+# block is trying to match, if applicable.
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/projectcalico/calico
+      tag: v${{package.version}}
+      expected-commit: 9ce0d6eae6f4c132526722d51821e2948aeaca25
+  - runs: |
+      go mod download
+  - working-directory: felix
+    pipeline:
+      # Equivalent to target: "build-bpf"
+      - runs: |
+          # TODO: this is a terrible hack to prevent the error:
+          #   /usr/include/gnu/stubs.h:7:11: fatal error: 'gnu/stubs-32.h' file not found
+          # I have no idea why it's wanting stubs-32.h and not correctly recognizing stubs-64.h instead
+          sed -i 's/#include <gnu\/stubs\.h>/\n/' /usr/include/features.h
```

Comment:
I'm not proud of this, and open to suggestions on how to do it the right way

Reply:
I think this is because its using the system headers when it should only be using the libbpf ones.